cider/exec/module/batch/CiderBatchUtils.cpp

```diff
@@ -149,6 +149,10 @@ int64_t getBufferNum(const ArrowSchema* schema) {
         case 's':
           return 1;
       }
+    case 't':
+      if (!strcmp(type, "tdm")) {
+        return 2;
+      }
```

Comment:
What about `if (strcmp(type, "tdm"))`?

Reply:
if 'str1' == 'str2', the return value will be 0,
if 'str1' < 'str2', the return value will less than 0,
if 'str1' > 'str2', the return value will more than 0.

Reply:
what does ```tdm``` mean?

Reply:
date64 in Arrow Type
![image](https://user-images.githubusercontent.com/110375636/196321817-948fd40c-b072-4637-ae69-5c1df0938df6.png)